templates/node/src/Payload.ts.twig

```diff
@@ -0,0 +1,59 @@
+import { readFileSync } from "fs";
```

Comment:
We can't import from `fs` or `path` here as these are node only, and this will break Next.js middleware

Reply:
What was the final deciison here?

Reply:
Even I don't have enough clarity. @loks0n please advise.

Reply:
We decided to use Blob in `fromFile` instead of path for node